server/datastore/mysql/app_configs_test.go

```diff
@@ -36,7 +35,7 @@ func TestAppConfig(t *testing.T) {
 		{"Backwards Compatibility", testAppConfigBackwardsCompatibility},
 		{"GetConfigEnableDiskEncryption", testGetConfigEnableDiskEncryption},
 		{"IsEnrollSecretAvailable", testIsEnrollSecretAvailable},
-		{"NDESSCEPProxyPassword", testNDESSCEPProxyPassword},
```

Comment:
Why is this test going away? (Maybe a mistake during merge conflict?)

Reply:
Wow, weird, good catch. Git actually auto-merged this. Not sure how it could have got that wrong.

Reply:
Oh, I looked and it's actually from my merge last week. The merge today was clean but that one last week had a conflict that I must have messed up.